src/formatters/default_formatter.erl

```diff
@@ -646,7 +646,12 @@ lay_no_comments(Node, Ctxt) ->
             Ctxt1 = reset_prec(Ctxt),
             D1 = beside(beside(lay_text_float("#"), lay(erl_syntax:record_expr_type(Node), Ctxt1)),
                         text("{")),
-            D2 = lay_fields(D1, erl_syntax:record_expr_fields(Node), Ctxt1, fun lay/2),
+            D2 = case erl_syntax:record_expr_fields(Node) of
+                [] ->
+                    lay_fields(D1, [], Ctxt1#ctxt{spaces_around_fields = false}, fun lay/2);
```

Comment:
```suggestion
                    par([D1,lay_text_float("}")]);
```
Didn't test it, but since there are no fields, it's probably what `lay_fields/4` will do, anyway.